examples/examples_test.go

```diff
@@ -79,21 +76,9 @@ func TestExamples(t *testing.T) {
 		},
 		{
 			Dir:          path.Join(cwd, "dynamic-provider/multiple-turns-2"),
-			Dependencies: []string{"@pulumi/pulumi"},
-		},
-		{
```

Comment:
The goal of this test was to ensure that when using the latest CLI with the 0.10.0 runtime, configuration still worked (because the 0.10.0 packages assumed that configuration keys would be passed as `foo:config:bar` instead of `foo:bar`).  However, since we were actually still linking in `@pulumi/pulumi` it wasn't actually doing what we expected.  I tried to retain the test but the 0.10.0 runtime didn't support more recent versions of node, and so it would fail because of that, so I just removed it from the test matrix